### ISSUE TITLE
Release 0.10.1 is Failing with ember-beta

### ISSUE THREAD

**Issue by jherdman** [RSSchermer/ember-multiselect-checkboxes#42]
The following error is raised with ember-beta

```
{ type: 'warn',
  text: '\'DEPRECATION: Using "_lookupFactory" is deprecated. Please use container.factoryFor instead. [deprecation id: container-lookupFactory] See TODO for more details.\\n        at logDeprecationStackTrace (http://localhost:7357/assets/vendor.js:24956:19)\\n        at HANDLERS.(anonymous function) (http://localhost:7357/assets/vendor.js:25067:7)\\n        at raiseOnDeprecation (http://localhost:7357/assets/vendor.js:24986:12)\\n        at HANDLERS.(anonymous function) (http://localhost:7357/assets/vendor.js:25067:7)\\n        at invoke (http://localhost:7357/assets/vendor.js:25083:7)\\n        at deprecate (http://localhost:7357/assets/vendor.js:25051:32)\\n        at Object.deprecate (http://localhost:7357/assets/vendor.js:36912:37)\\n        at Container.lookupFactory (http://localhost:7357/assets/vendor.js:20485:19)\\n        at Class._lookupFactory (http://localhost:7357/assets/vendor.js:52170:33)\'\n' }
```

This problem appears to be resolved with the master branch of this add-on. Could we therefore have a release?

**Comment by RSSchermer** [comment]
Yes, releasing a new patch version is on my todo list. I'll see about doing so tonight.

**Comment by jherdman** [comment]
Thank you so much! 😍

**Comment by RSSchermer** [comment]
Published as 0.10.2.
